server/server.go

```diff
@@ -66,6 +67,7 @@ type config struct {
 	Port             string `envconfig:"PORT" default:"6789"`
 	ReportPath       string `envconfig:"REPORT_PATH" default:""`
 	PathPrefix       string `envconfig:"KEPLOY_PATH_PREFIX" default:"/"`
+	LogPath          string `envconfig:"LOG_PATH"`
```

Comment:
Is this path for logs is absolute? Can it check for relative path and resolve to absolute path.

Reply:
Indeed, it will create a file in the same dir as that of go.mod file. I have been using this at my work place as well and it has been working fine.

Reply:
LGTM. Please add comments for the format of the logPath.

Reply:
Sure 👍